cwms-data-api/src/test/java/cwms/cda/api/CountyControllerTestIT.java

```diff
@@ -61,4 +61,58 @@ void test_county_catalog() {
             .body("[0].county-id", equalTo("000"))
             .body("[0].state-initial", equalTo("00"));
     }
+
+    @Test
+    void test_county_catalog_default_content_type() {
+        given()
+            .log().ifValidationFails(LogDetail.ALL,true)
+            .accept(Formats.DEFAULT)
+            .contentType(Formats.DEFAULT)
```

Comment:
Is contentType needed here?   I would think that is only needed on end-points that involve the client sending a request body.  I wouldn't think its needed on GET.

Reply:
We probably aren't great about doing this right in the all the IT.  You probably copied this from somewhere else.  I'm wondering if we should go thru all the tests and make sure there aren't extra cruft

Reply:
I removed the contenttype part, and re-tested, it works fine for our use case.
As for where I copied it from, the existing test there `test_county_catalog()` is what I ended up using as the framework.